executor/executor_test.go

```diff
@@ -1332,10 +1332,10 @@ func (s *testSuite) TestGeneratedColumnRead(c *C) {
 	result.Check(testkit.Rows(`1 2 3 2`, `3 4 7 12`, `8 8 16 64`))
 
 	// Test generated column in subqueries.
-	result = tk.MustQuery(`SELECT * FROM test_gc_read t WHERE t.a not in (SELECT t.a FROM test_gc_read t where t.c > 5)`)
+	result = tk.MustQuery(`SELECT * FROM test_gc_read t WHERE t.a not in (SELECT t.a FROM test_gc_read t where t.c > 5) order by a`)
```

Comment:
We can right as `result.Sort().Check`

Reply:
this involves many result order changes, adding a `order by` in the tail of a query is the simplest way to make test happy.

Reply:
But this may change the plan of query...